content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -21,13 +21,18 @@ To install and configure Istio in a Kubernetes cluster, follow these instruction
 
 ## Installation steps
 
-To install Istio's core components you can choose one of the following four
-**mutually exclusive** options.
+1. Install Istio's [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the kube-apiserver:
 
-However, we recommend you to install with the
-[Helm Chart](/docs/setup/kubernetes/helm-install/) for production
-installations of Istio. With this installation, you can leverage all the
-options to configure and customize Istio to your needs.
+    {{< text bash >}}
+    $ kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml -n istio-system
+    {{< /text >}}
+
+1. To install Istio's core components you can choose any of the following four
+**mutually exclusive** options described below. However, for a production setup of Istio,
+we recommend installing with the
+[Helm Chart](/docs/setup/kubernetes/helm-install/), to leverage all the
+configuration options and to customize Istio to your needs.
```

Comment:
options.  This will permit customization of Istio to operator specific needs.

Reply:
ok.